Kevin-Hernandez.txt

```diff
@@ -0,0 +1,85 @@
+https://codepen.io/KevinH1/full/pogZRLv
+Here is a new file. Yay!
+
+1. What is Semantic HTML? describes its own meaning.
+
+    2. What is HTML used for? It's used for documents to be displayed in a browser. 
+
+    3. What is an attribute and where do we put it? provides addtional info about elements. Used when adding a link to something.
+
+    4. What is the h1 tag used for? How many times should I use it on a page? H1 is used for the most important "title" on the page. It can only be used once.
+
+    5. Name two tags that have required attributes <img> and <a>
+
+    6. What do we put in the head of our HTML document? <html> tag
+
+    7. What is an id? Its used to name an element
+
+    8. What elements can I add an id to? All of them. Its part of the Global Attributes
+
+    9. How many times can I use the same id on a page? Once
+
+    10. What is a class? specifies one or more classnames for an element
+
+    11. What elements can I add a class to? All of them
+
+    12. How many times can I use the same class on a page? Unlimited
+
+    13. How do I get my link to open in a new tab? just by adding target="_blank"
+
+    14. What is the alt attribute in the image tag used for? used for an alternate text if the img doesn't work
+
+    15. How do I reference an id? # then the name of the id
+
+    16. What is the difference between a section and a div? Section offers importance while div does not
+
+    17. What is CSS used for? for styling a webpage
+
+    18. How to we select an element? Example - every h2 on the page. by putting h2 then what you want to change
+
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one. A class can be used to select more than one element. An id is used to select only one element.
+ex. one use and id for something very unique like a h1 heading (since there's only 1 of them) and use a class for <div> since you can use it muliple times
+
+    20. How do we select classes in CSS? ."then the class name"
+
+    21. How do we select a p element with a single class of “human””? .human p
```

Comment:
This will select a `<p>` element that is a descendant of an element with the class `human`. How would you select the following?
```html
<p class="human">Hello, world!</p>
```